pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -153,9 +150,7 @@ var (
 		"periodic-ci-openshift-multiarch-master-nightly-4.9-upgrade-from-nightly-4.8-ocp-remote-libvirt-s390x",
 		"periodic-ci-openshift-release-master-ci-4.9-upgrade-from-stable-4.8-e2e-aws-uwm",
 		"periodic-ci-openshift-release-master-ci-4.9-upgrade-from-stable-4.8-e2e-gcp-ovn-upgrade",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-ovn-local-gateway",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-azure-fips",
-		"periodic-ci-openshift-release-master-nightly-4.10-e2e-gcp-fips",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-gcp-ovn-upgrade",
```

Comment:
isn't this used for payload promotion?

Reply:
Only through aggregation.  The jobs themselves have dismal overall pass rates, they belong in never-stable. 

never-stable doesn't mean we forget about them, but it does make it easier for us to find new issues.  It may not make it easier for you to find problems, but it does for me.